### ISSUE TITLE
论文与模型

### ISSUE THREAD

**Issue by chenghao-matrix** [eezkni/FDL#8]
尊敬的作者你好， 论文的工作真的非常好，提出了一个非常新颖的损失函数，彻底颠覆了很多应用领域的损失函数设计方法。研究论文后，感觉还是理解的不够深刻。对于下面代码红色箭头部分：我没理解这里为什么是self.model = VGG or ResNet等？？不同的训练任务都要用这些模型吗？比如，我的任务是图像增强，图像匹配，超分重建等任务，那这里的self.model该怎么确定？ 其实就是我没懂我怎么把你的损失函数应用到我的任务中？感谢 作者！

<img width="877" height="758" alt="Image" src="https://github.com/user-attachments/assets/b4bd44cd-8a5d-41a6-8b91-ec3cbb3f840f" />

**Comment by Chtholly17** [comment]
你好，model用于提取图像特征，默认使用预训练的VGG网络。使用默认值就可以

**Comment by cxzhou35** [comment]
> 你好，model用于提取图像特征，默认使用预训练的VGG网络。使用默认值就可以

Hi, can I use the vae encoded latents as the inputs for the FDL loss?

Example like this:

```python
x_gt = vae.encode(image_gt)
x_pred = model(noise)

loss = FDL_loss(x_gt, x_pred)
```
